packages/stargate/src/stargateclient.ts

```diff
@@ -94,6 +96,7 @@ export interface PrivateStargateClient {
 
 export class StargateClient {
   private readonly tmClient: TendermintClient;
+  private readonly queryClient: QueryClient & BankExtension;
```

Comment:
Why is this private?

The QueryClient is beautifully extensible, but now we need to explicitly add mappings like `getBalance` below to use it?

I guess we need that for the typings? QueryClient gives us raw data from grpc, and this converts it to some types we control?

Reply:
Exactly, the types from the autogenerated protobuf are a pain to work with. You don't want to throw `number | Long | null | undefined` on the user for a field that is non-nullable.